src/com/dotmarketing/portlets/fileassets/business/FileAsset.java

```diff
@@ -87,7 +95,7 @@ public long getFileSize() {
 	public int getHeight() {
 		int height = 0;
 		try {
-			height = javax.imageio.ImageIO.read(getFileAsset()).getHeight();
+			height = ImageUtil.getInstance().getDimension(getFileAsset()).height;
```

Comment:
I think we modify the code a little to call one time ImageUtil.getInstance().getDimension(getFileAsset()) and store the dimension in a private variable, so that way you don't load the file twice to get height or width in the same instance.
Just check if dimension is not null and get the proper value